### ISSUE TITLE
Kedro Viz throws an error on the experiment tab	

### ISSUE THREAD

**Issue by maddataanalyst** [kedro-org/kedro-viz#1276]
## Description

I have created a trivial project from scratch, using the latest version of Kedro and Kedro Viz. Coding on WSL2 Ubuntu.
A trivial function is logging some fake, constant metric. Session_store.db is created correctly, metrics.json file as well. When opening the "experiments" tab in Kedro-Viz, I get the error in the log and console:

`'str' object has no attribute 'get'                                                                                      execute.py:153

                             GraphQL request:2:3
                             1 | query getRunsList {
                             2 |   runsList {
                               |   ^
                             3 |     bookmark
                             Traceback (most recent call last):`

## Context

Trivial, toy project to play with experiment tracking in Kedro crashes Kedro-viz when opening 'experiments tab' making it unusable.

![image](https://user-images.githubusercontent.com/33813997/222442708-a3fc7000-24d1-4ab6-99f9-677cc3a37289.png)
![image](https://user-images.githubusercontent.com/33813997/222440121-a3f81810-02d3-4502-8d48-f5b98406b912.png)
![image](https://user-images.githubusercontent.com/33813997/222440204-82c6759c-f613-4ca1-8488-b842405abe2a.png)

## Steps to Reproduce

1. Kedro-viz configured in settings.py as 

```from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
from pathlib import Path

SESSION_STORE_CLASS = SQLiteStore
SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
```

2. Single function in nodes returning constant metric:

```
def doit():
    return {"a": 1.12, "b": 32.5}
```

3. Pipeline definition:
```
from kedro.pipeline import Pipeline, node, pipeline
from .nodes import doit

def create_pipeline(**kwargs) -> Pipeline:
    return pipeline([
        node(
            func=doit,
            inputs=[],
            outputs='metrics',
            name='check_metrics_log'
        )
    ])
```
4. Conf/base/catalog.yml:

```
metrics:
  type: tracking.MetricsDataSet
  filepath: data/09_tracking/metrics.json
  layer: experiment
```
5. Execute `kedro run --nodes check_metrics_log` in cmd.
6. Start Kedro-Viz in project.

## Expected Result

After opening experiments tab, the "Experiments" dashboard should show up.

## Actual Result

The exception thrown by the Kedro-viz (stack trace):

```
2023-03-02 14:31:32,125 - strawberry.execution - ERROR - 'str' object has no attribute 'get'

GraphQL request:2:3
1 | query getRunsList {
2 |   runsList {
  |   ^
3 |     bookmark
Traceback (most recent call last):
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/graphql/execution/execute.py", line 521, in execute_field
    result = resolve_fn(source, info, **args)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/schema/schema_converter.py", line 532, in _resolver
    return _get_result(_source, strawberry_info, **kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/schema/schema_converter.py", line 524, in _get_result
    return field.get_result(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/field.py", line 177, in get_result
    return self.base_resolver(*args, **kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/types/fields/resolver.py", line 187, in __call__
    return self.wrapped_func(*args, **kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/api/graphql/schema.py", line 66, in runs_list
    return format_runs(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/api/graphql/serializers.py", line 66, in format_runs
    return [
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/api/graphql/serializers.py", line 67, in <listcomp>
    format_run(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/api/graphql/serializers.py", line 46, in format_run
    run_command=run_blob.get("cli", {}).get("command_path"),
AttributeError: 'str' object has no attribute 'get'
Stack (most recent call last):
  File "/home/myuser/anaconda3/envs/kedroenv/bin/kedro", line 8, in <module>
    sys.exit(main())
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro/framework/cli/cli.py", line 211, in main
    cli_collection()
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro/framework/cli/cli.py", line 139, in main
    super().main(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/launchers/cli.py", line 134, in viz
    run_server(**run_server_kwargs)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/kedro_viz/server.py", line 103, in run_server
    uvicorn.run(app, host=host, port=port, log_config=None)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/main.py", line 569, in run
    server.run()
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/protocols/http/httptools_impl.py", line 419, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/middleware/message_logger.py", line 82, in __call__
    await self.app(scope, inner_receive, inner_send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/fastapi/applications.py", line 270, in __call__
    await super().__call__(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/routing.py", line 706, in __call__
    await route.handle(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/asgi/__init__.py", line 54, in __call__
    await self.http_handler_class(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/asgi/handlers/http_handler.py", line 55, in handle
    response = await self.get_http_response(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/asgi/handlers/http_handler.py", line 153, in get_http_response
    result = await execute(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/asgi/handlers/http_handler.py", line 206, in execute
    return await self.schema.execute(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/schema/schema.py", line 232, in execute
    result = await execute(
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/strawberry/schema/execute.py", line 153, in execute
    process_errors(result.errors, execution_context)
2023-03-02 14:32:07,838 - uvicorn.error - ERROR - Traceback (most recent call last):
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/starlette/routing.py", line 674, in lifespan
    await receive()
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/middleware/message_logger.py", line 62, in inner_receive
    message = await receive()
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/site-packages/uvicorn/lifespan/on.py", line 137, in receive
    return await self.receive_queue.get()
  File "/home/myuser/anaconda3/envs/kedroenv/lib/python3.9/asyncio/queues.py", line 166, in get
    await getter
asyncio.exceptions.CancelledError
```

## Your Environment

Python 3.9, below the output from `pip list`

Package                Version
---------------------- ---------
aiofiles               22.1.0
antlr4-python3-runtime 4.9.3
anyconfig              0.10.1
anyio                  3.6.2
argon2-cffi            21.3.0
argon2-cffi-bindings   21.2.0
arrow                  1.2.3
asttokens              2.2.1
attrs                  22.2.0
Babel                  2.12.1
backcall               0.2.0
beautifulsoup4         4.11.2
binaryornot            0.4.4
black                  22.12.0
bleach                 6.0.0
build                  0.10.0
cachetools             5.3.0
certifi                2022.12.7
cffi                   1.15.1
chardet                5.1.0
charset-normalizer     3.0.1
click                  8.1.3
comm                   0.1.2
cookiecutter           2.1.1
coverage               7.2.1
debugpy                1.6.6
decorator              5.1.1
defusedxml             0.7.1
dynaconf               3.1.11
exceptiongroup         1.1.0
executing              1.2.0
fastapi                0.89.1
fastjsonschema         2.16.3
flake8                 4.0.1
fsspec                 2023.1.0
gitdb                  4.0.10
GitPython              3.1.31
graphql-core           3.2.3
greenlet               2.0.2
h11                    0.14.0
httptools              0.5.0
idna                   3.4
importlib-metadata     6.0.0
importlib-resources    5.12.0
iniconfig              2.0.0
ipykernel              6.21.2
ipython                8.11.0
ipython-genutils       0.2.0
ipywidgets             8.0.4
isort                  5.12.0
jedi                   0.18.2
Jinja2                 3.1.2
jinja2-time            0.2.0
jmespath               0.10.0
json5                  0.9.11
jsonschema             4.17.3
jupyter                1.0.0
jupyter_client         8.0.3
jupyter-console        6.6.2
jupyter_core           5.2.0
jupyter-server         1.23.6
jupyterlab             3.5.3
jupyterlab-pygments    0.2.2
jupyterlab_server      2.15.2
jupyterlab-widgets     3.0.5
kedro                  0.18.5
kedro-telemetry        0.2.3
kedro-viz              5.3.0
markdown-it-py         2.2.0
MarkupSafe             2.1.2
matplotlib-inline      0.1.6
mccabe                 0.6.1
mdurl                  0.1.2
mistune                2.0.5
more-itertools         9.1.0
mypy-extensions        1.0.0
nbclassic              0.5.2
nbclient               0.7.2
nbconvert              7.2.9
nbformat               5.7.3
nbstripout             0.6.1
nest-asyncio           1.5.6
networkx               3.0
notebook               6.5.2
notebook_shim          0.2.2
numpy                  1.24.2
omegaconf              2.3.0
orjson                 3.8.7
packaging              23.0
pandas                 1.5.3
pandocfilters          1.5.0
parso                  0.8.3
pathspec               0.11.0
pexpect                4.8.0
pickleshare            0.7.5
pip                    22.3.1
pip-tools              6.12.3
platformdirs           3.0.0
plotly                 5.13.1
pluggy                 1.0.0
prometheus-client      0.16.0
prompt-toolkit         3.0.38
psutil                 5.9.4
ptyprocess             0.7.0
pure-eval              0.2.2
pycodestyle            2.8.0
pycparser              2.21
pydantic               1.10.5
pyflakes               2.4.0
Pygments               2.14.0
pyproject_hooks        1.0.0
pyrsistent             0.19.3
pytest                 7.2.1
pytest-cov             3.0.0
pytest-mock            1.13.0
python-dateutil        2.8.2
python-dotenv          1.0.0
python-slugify         8.0.1
pytoolconfig           1.2.5
pytz                   2022.7.1
PyYAML                 6.0
pyzmq                  25.0.0
qtconsole              5.4.0
QtPy                   2.3.0
requests               2.28.2
rich                   13.3.1
rope                   1.7.0
semver                 2.13.0
Send2Trash             1.8.0
setuptools             65.6.3
six                    1.16.0
smmap                  5.0.0
sniffio                1.3.0
soupsieve              2.4
SQLAlchemy             1.4.46
stack-data             0.6.2
starlette              0.22.0
strawberry-graphql     0.159.0
tenacity               8.2.2
terminado              0.17.1
text-unidecode         1.3
tinycss2               1.2.1
toml                   0.10.2
tomli                  2.0.1
toposort               1.10
tornado                6.2
traitlets              5.9.0
typing_extensions      4.5.0
urllib3                1.26.14
uvicorn                0.20.0
uvloop                 0.17.0
watchfiles             0.18.1
watchgod               0.8.2
wcwidth                0.2.6
webencodings           0.5.1
websocket-client       1.5.1
websockets             10.4
wheel                  0.38.4
widgetsnbextension     4.0.5
zipp                   3.15.0

**Comment by AntonyMilneQB** [comment]
Hi @maddataanalyst and thanks very much for this bug report. Sorry for the difficulty you encountered - this is a known issue with Kedro 0.18.5 and should be fixed in Kedro 0.18.6 which will be released shortly.

For the time being everything should correctly if you use Kedro 0.18.4. So I would recommend:
- do `pip uninstall kedro && pip install kedro==0.18.4` to downgrade
- assuming you’re just playing around and don’t have any run information that you care about, delete your data/session_store.db file
- you can keep the rest of the project as it is
- all should then work as expected 🎉

**Comment by maddataanalyst** [comment]
Hello!
Thank you very much. The described fix indeed resolves the problem. I will be waiting for the new release before upgrading again.
Thank you very much and keep up the good work.
